sdcm/sct_events.py

```diff
@@ -638,6 +638,9 @@ def _load_from_raw_alert(self,  # pylint: disable=too-many-arguments,invalid-nam
             self.alert_name = self.labels.get('alertname', '')
         else:
             self.alert_name = ''
+        severity = self.labels.get('severity')
```

Comment:
per our discussion, the default should be warning and custom configured where we explicitly set severity we should use it

Reply:
fixed